Sources/ConcentricOnboarding/Helpers/AnimatableShape.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  AnimatableShape.swift
+//  ConcentricOnboarding
+//
+//  Created by Daniil Manin on 21.09.2021.
+//  Copyright © 2021 Exyte. All rights reserved.
+//
+
+import SwiftUI
+
+struct AnimatableShape: Shape {
+	
+	enum AnimationType {
```

Comment:
Usually you use spaces, but here they are tabs - please pick one (I prefer spaces))